src/bin/cargo-build-all-features.rs

```diff
@@ -1,6 +0,0 @@
-use cargo_all_features::{run, test_runner::CargoCommand};
```

Comment:
issue: we shouldn't remove the old commands, they should be kept around for backcompat